### PR TITLE
Feature/aqua creator components

### PR DESCRIPTION
**Pull request by ShanghaiG**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -26,6 +27,8 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-firebase-hooks": "^5.1.1",
+    "react-icons": "^4.8.0",
+    "react-router": "^6.11.2",
```
[the remark this answers is not in the document]
Comment (ShanghaiG) [reply frontend/package.json:31]: React icons are used, react-router is not longer used

File: frontend/src/app/(authorized)/creator/AquaSummaryPage.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  aquariumDimensions,
+  aquariumData,
+}: AquaSummaryPageProps) => {
+  console.log("aquariumName", aquariumName);
+  return (
+    <div className="my-10 px-1 md:px-20 flex flex-col items-center">
+      <input
+        value={aquariumName}
+        onChange={(e) => setAquariumName(e.target.value)}
+        className="text-3xl font-bold bg-transparent border-none outline-none mb-2 w-1/5 text-center"
```
[the remark this answers is not in the document]
Comment (ShanghaiG) [reply frontend/src/app/(authorized)/creator/AquaSummaryPage.tsx:26]: image
suseg

File: frontend/tailwind.config.js [hunk 3]
```diff
@@ -1,17 +1,45 @@
...
+    },
     extend: {
       colors: {
         primary: "#3056D3",
         background: "#F4F7FF",
-      }
-    }
+        stepsGreen: "#26C196",
+      },
+      transform: ["hover", "focus"],
```
[the remark this answers is not in the document]
Comment (ShanghaiG) [reply frontend/tailwind.config.js:28]: I decided to drop that one, forgot to clear it here, thanks

File: frontend/src/app/(authorized)/creator/AquaSummaryPage.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (jowczar) [reply frontend/src/app/(authorized)/creator/AquaSummaryPage.tsx:26]: Let's try it again
image

File: frontend/src/components/Modal/index.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  cancelButtonText,
+  detailsButtonText,
+  onCancelClick,
+  onDetailsClick,
+}: any) => {
+  return (
+    <>
+      <div className="fixed top-0 left-0 flex h-full min-h-screen w-full items-center justify-center  px-4 py-5 ">
+        <div
+          onClick={(e) => e.stopPropagation()}
```
[the remark this answers is not in the document]
Comment (ShanghaiG) [reply frontend/src/components/Modal/index.tsx:15]: No longer needed, fixed in other place

**Review by jowczar — APPROVED** [review]

**Pull request merged**